.pre-commit-config.yaml

```diff
@@ -25,8 +25,13 @@ repos:
       - id: flake8
         args: ['--count', '--select', 'E101,W191,W291,W292,W293,W391,E111,E112,E113,E303,E304,E306,E502,E722,E901,E902,F822,F823']
 
+#  - repo: https://github.com/asottile/seed-isort-config
+#    rev: v2.1.0
+#    hooks:
+#      - id: seed-isort-config
+
   - repo: https://github.com/timothycrosley/isort
-    rev: 4.3.21-2
+    rev: 4.3.21
```

Comment:
Why was this changed?

Reply:
Sorry @nabobalis I was testing something. Have just reverted the change now.